blueocean-github-pipeline/src/test/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmContentProviderTest.java

```diff
@@ -24,6 +27,8 @@
 /**
  * @author Vivek Pandey
  */
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore({"javax.crypto.*", "javax.security.*", "javax.net.ssl.*", "com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*", "org.w3c.dom.*"})
```

Comment:
Will javac let us use a static final array to init an annotation, or does it insist on a literal? Seems updating all of these in 6 months when the list changes slightly for J12 will be a PITA

Reply:
Where would you define it? I was thinking last night that maybe we could just Subclass PowerMockIgnore to BluePowerMockIgnore and just pass in any extras ontop of the defaults.

Reply:
If we can get away with it using a static final list seems like the safer option. Where it's defined doesn't really matter too much to me. But we'd have a single place to change it, a name, and also a place to document why things are/aren't required to be explicitly ignore.